devimint/src/federation.rs

```diff
@@ -30,16 +32,59 @@ impl Federation {
     pub async fn new(
         process_mgr: &ProcessManager,
         bitcoind: Bitcoind,
-        vars: BTreeMap<usize, vars::Fedimintd>,
+        servers: usize,
+        // vars: BTreeMap<usize, vars::Fedimintd>,
     ) -> Result<Self> {
         let mut members = BTreeMap::new();
-        for (peer, var) in &vars {
+        let mut vars = BTreeMap::new();
+
+        // FIXME: this is a hack to pass fed.server_gen_params in
```

Comment:
@justinmoon do you think this is still something to be fixed? what would be the proper way of doing it?